CRM/Contribute/Import/Form/MapField.php

```diff
@@ -162,50 +130,25 @@ public function buildQuickForm() {
    *   list of errors to be posted back to the form
    */
   public static function formRule($fields, $files, $self) {
-    $errors = [];
-    $fieldMessage = NULL;
-    $contactORContributionId = $self->isUpdateExisting() ? 'contribution_id' : 'contribution_contact_id';
-    if (!array_key_exists('savedMapping', $fields)) {
```

Comment:
This check doesn't make sense - the mapping needs to be valid regardless of whether a saved mapping was loaded.

I did a check & you can still go back even when you can't go forwards due to validation

![image](https://user-images.githubusercontent.com/336308/188352365-d05ddc37-8221-4bc2-8d72-81b913fdcc5b.png)